### ISSUE TITLE
Stream deck mini not recognised by Companion

### ISSUE THREAD

**Issue by ErwinVisschedijk** [bitfocus/companion#951]
My brand new Stream deck mini is not recognised by Companion.
It works well in the Elgato Stream deck app.
When I start Elgato Stream deck app, the logo appears only in the middle button.
Then I close the app and the logo is spread out over 9 buttons.
I start Companion and run it on: http://127.0.0.1:8000/
In the Companion Admin I see under 'Surfaces' only the emulator. When I click Rescan USB, it blinks and no Stream deck is found.
What did I mis?

**Comment by JeffreyDavidsz** [comment]
Are you using version 2.0? Version 1.4 was released -I think- before the mini file was created

**Comment by ErwinVisschedijk** [comment]
I am using Companion 1.4. Maybe I used the wrong name for the Stream deck. It is the one with 15 buttons.

**Comment by JeffreyDavidsz** [comment]
Yeah Streamdeck mini is a 6-button I think. This year a new version came out of the 15-button one that created what you are seeing. We (Hakon) fixed this, but you would need to go to version 2.0

**Comment by ErwinVisschedijk** [comment]
Does 2.0 already supports ATEM switchers? And where to download version 2.0?

**Comment by JeffreyDavidsz** [comment]
Yes it does, you can download it from the userarea (https://bitfocus.io/user/downloads) and clink on the small 'builds' link

**Comment by ErwinVisschedijk** [comment]
Thanks a lot. It works with 2.0

**Comment by haakonnessjoen** [comment]
@JeffreyDavidsz: Mini support was added in companion 1.2.0

But since you do not have the mini, but the 15 button regular stream deck. There are newer versions of the hardware that is only supported in 2.0

**Comment by MarkG40** [comment]
Where do you go to download version 2.0? I also have the 15 button stream deck and it's not being recognized

**Comment by JeffreyDavidsz** [comment]
Check user area on the website you'll find 2.0 beta

**Comment by rontomasch** [comment]
> My brand new Stream deck mini is not recognised by Companion.
> It works well in the Elgato Stream deck app.
> When I start Elgato Stream deck app, the logo appears only in the middle button.
> Then I close the app and the logo is spread out over 9 buttons.
> I start Companion and run it on: http://127.0.0.1:8000/
> In the Companion Admin I see under 'Surfaces' only the emulator. When I click Rescan USB, it blinks and no Stream deck is found.
> What did I mis?



I believe I found the solution. In the Bitfocus companion software, click on the surfaces tab. Mine had, Elgato Streamdeck Emulator & Elgato Streamdeck Plugin. What was missing was the Elgato Streamdeck device, Make sure the stream deck software is closed. Go to the system tray and right click on the stream deck software icon and click on, "Quit Stream Deck". Now go back to the surfaces tab in companion software and click, "Rescan USB". This should appear; Under the "ID" should be a series of numbers and under "Type" should be listed, "Elgato Streamdeck Device". Once these things appeared I was able to load all the shortcuts I wanted both from vMix and Blackmagic Design ATEM mini Pro. Everything works as expected now.
